modules/openapi-generator/src/main/resources/python/model_utils.mustache

```diff
@@ -1211,13 +1211,19 @@ def model_to_dict(model_instance, serialize=True):
                 # exist in attribute_map
                 attr = model_instance.attribute_map.get(attr, attr)
             if isinstance(value, list):
-                if not value or isinstance(value[0], PRIMITIVE_TYPES):
-                    # empty list or primitive types
-                    result[attr] = value
-                elif isinstance(value[0], ModelSimple):
-                    result[attr] = [x.value for x in value]
-                else:
-                    result[attr] = [model_to_dict(x, serialize=serialize) for x in value]
+               if not value:
+                   # empty list or None
+                   result[attr] = value
+               else:
+                   res = []
+                   for v in value:
```

Comment:
This fails to handle the case where `v` is a `dict`, i.e. a free-form object.

e.g. a spec like:

```json
        "type": "array",
        "items": {
          "type": "object",
          "additionalProperties": true
        },
```

Reply:
That could be a bug related to inline schema handling.
Is there a model made and used for that schema?
If so, we handle it with line 1225
Can you want to post an issue describing the problem that you've found? Or a PR fixing it?

Reply:
Good points, I'm not sure if the codegen is wrong or if the `model_to_dict` is just missing the specific case.

The generated client code I'm experiencing this issue with is: https://github.com/sajari/sdk-python/blob/6bf9c20e4a2639eaf9774d1770b6a7ac39ed33c0/sajari_client/model/batch_upsert_records_request.py#L130-L131

e.g. when I attempt to serialise my request like:
```python
        request = BatchUpsertRecordsRequest(
            pipeline=BatchUpsertRecordsRequestPipeline(name="app", version="1"),
            records=BatchUpsertRecordsRequestRecords([{"foo": "bar"}]),
            variables={},
        )
        model_to_dict(request, serialize=True)
```

it fails with:
```
Traceback (most recent call last):
  File "/Users/jsok/dev/github.com/sajari/sdk-python/test/test_reproduction.py", line 83, in test_request_serialisation
    print(model_to_dict(request, serialize=True))
  File "/Users/jsok/dev/github.com/sajari/sdk-python/sajari_client/model_utils.py", line 1693, in model_to_dict
    res.append(model_to_dict(v, serialize=serialize))
  File "/Users/jsok/dev/github.com/sajari/sdk-python/sajari_client/model_utils.py", line 1665, in model_to_dict
    if model_instance._composed_schemas:
AttributeError: 'dict' object has no attribute '_composed_schemas'
```

Happy to contribute a PR that makes the fix in the `model_to_dict` method if that's the right place for it.